### ISSUE TITLE
Plot data

### ISSUE THREAD

**Issue by jwunderl** [iEvilMango/INFO201FinalProjectAE2#5]
Create file with functions to plot data in unique / useful ways for output

**Comment by BrydenR** [comment]
I don't think that we will actually be using a plot for data here, so I am closing the issue.

**Comment by BrydenR** [comment]
Reopening as a generic display. May include a word chart, or something tbd.
